### ISSUE TITLE
keras version

### ISSUE THREAD

**Issue by OlDor** [sciencefans/CaffeMex_v2#10]
Caffe doesn't seem to be the most popular ML library today.
Could you please release a keras, tensorflow or pytorch demo version of the COCO loss layer?
